### DISCUSSION TITLE
failed to connect to bootstrap peer

### DISCUSSION THREAD

**Discussion by nihaoqingtuan** [filecoin-project/lotus#10432] [General Q&A]

**Comment by rjan90** [comment]
Hey @nihaoqingtuan!

Are you able to sync? Can you check if `lotus sync wait`. You might fail to dial some of the bootstrap peers since they might be across the globe from where you are.

So just trying find out if you are able to connect to some bootstrappers and sync, or if you are failing completly to sync?
